modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/FragmentEntryFragmentRenderer.java

```diff
@@ -229,7 +230,7 @@ private String _renderFragmentEntry(
 		FragmentRendererContext fragmentRendererContext, String html,
 		HttpServletRequest httpServletRequest) {
 
-		StringBundler sb = new StringBundler(25);
+		StringBundler sb = new StringBundler(26);
```

Comment:
more of just a noob question here... why did you increment this? and how did you know to increment it?

Reply:
That is done automatically by SF. I assume it counts the number of strings we are concatenating and uses that to write that number.

Reply:
![eric-wareheim-mind-blown-gif-by-tim-and-eric](https://github.com/liferay-frontend/liferay-portal/assets/6843530/1300449f-3fdb-47ac-bd97-c33372a83f3e)